### ISSUE TITLE
duplicateImports rule wiped out @testable import

### ISSUE THREAD

**Issue by 2bjake** [nicklockwood/SwiftFormat#575]
A minor issue, but it looks like ```duplicateImports``` rule doesn't watch out for ```@testable```. A test file with imports like this:

```swift
import XCTest
import MyService
@testable import MyApp
@testable import MyService
```

got formatted like this:

```swift
@testable import MyApp
import MyService
import XCTest
```

which caused the test file to fail to build. I would have expected:

```swift
@testable import MyApp
@testable import MyService
import XCTest
```

**Comment by nicklockwood** [comment]
@2bjake thanks for reporting - I'll get it fixed asap.

**Comment by nicklockwood** [comment]
@2bjake fixed in 0.44.5
